catalogs/catalog-hive/src/test/java/org/apache/gravitino/catalog/hive/integration/test/CatalogHiveIT.java

```diff
@@ -629,6 +630,9 @@ public void testListTables() {
     catalog
         .asTableCatalog()
         .createTable(hudiTable, createColumns(), null, ImmutableMap.of("provider", "hudi"));
+    catalog
+        .asTableCatalog()
+        .createTable(hiveTable, createColumns(), null, ImmutableMap.of("provider", "hive"));
     NameIdentifier[] tables = catalog.asTableCatalog().listTables(Namespace.of(schemaName));
     Assertions.assertEquals(0, tables.length);
```

Comment:
The expected table number needs to be updated to 1

Reply:
I see.